lib/shared/config.rb

```diff
@@ -0,0 +1,48 @@
+require 'json'
+
+require_relative 'errors/file_not_found'
+
+module GoogleDrive
```

Comment:
Put this file under "lib/google_drive" directory. I try to make file path and class hierarchy (GoogleDrive::Config) consistent.